src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -1786,10 +1807,32 @@ protected String extractAuthorName(Document doc) {
                 }
             }
 
+            // globalbankingandfinance.com
+            if (authorName.isEmpty()) {
+                authorName = SHelper.innerTrim(doc.select("div[class=the-content post-content clearfix] p strong em").text());
+                if(DEBUG_AUTHOR_EXTRACTION && !authorName.isEmpty()) System.out.println("AUTHOR: div[class=the-content post-content clearfix] p strong em");
+            }
+
+            // fortune.com
+            if (authorName.isEmpty()) {
+                authorName = SHelper.innerTrim(doc.select("head meta[property=author]").attr("content"));
+                if(DEBUG_AUTHOR_EXTRACTION && !authorName.isEmpty()) System.out.println("AUTHOR: head meta[property=article:author]");
+            }
+
             if (authorName.isEmpty()) {  // for "opengraph"
```

Comment:
move general cases like open graph, twitter, etc to the top. have site specific ones afterwards